src/main/groovy/com/avioconsulting/mule/linter/rule/pom/PomPropertyValueRule.groovy

```diff
@@ -0,0 +1,43 @@
+package com.avioconsulting.mule.linter.rule.pom
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+
+class PomPropertyValueRule extends Rule {
+
+    static final String RULE_ID = 'MAVEN_PROPERTY'
+    static final String RULE_NAME = 'maven property match'
+    static final String RULE_VIOLATION_MESSAGE = ' maven property does not match or exists'
+
+    private final String propertyName
+    private final String propertyValue
+
+    PomPropertyValueRule(String propertyName, String propertyValue) {
+        this.ruleId = RULE_ID
+        this.ruleName = RULE_NAME
+        this.propertyName = propertyName
+        this.propertyValue = propertyValue
+    }
+
+    PomPropertyValueRule(String ruleId, String ruleName, String propertyName, String propertyValue) {
+        this.ruleId = ruleId
+        this.ruleName = ruleName
+        this.propertyName = propertyName
+        this.propertyValue = propertyValue
+    }
+
+    @Override
+    List<RuleViolation> execute(Application app) {
+        List<RuleViolation> violations = []
+
+        String pomPropertyValue = app.pomFile.getProperty(propertyName)
+        if (!pomPropertyValue.equalsIgnoreCase(propertyValue)) {
+            violations.add(new RuleViolation(this, app.pomFile.getFile().absolutePath,
+                    0, propertyName + RULE_VIOLATION_MESSAGE))
```

Comment:
Need actual line from pom.xml where the property exists for the violation, unless it doesn't exist.  We should probably default to where the properties block is in that case

Reply:
Maven library is parsing the xml, so we dont know the actual line number where it should belong.